jiant/proj/main/modeling/taskmodels.py

```diff
@@ -333,23 +346,34 @@ def get_output_from_encoder(encoder, input_ids, segment_ids, input_mask) -> Enco
         return EncoderOutput(pooled=pooled, unpooled=unpooled)
 
 
-def get_output_from_standard_transformer_models(encoder, input_ids, segment_ids, input_mask):
-    output = encoder(input_ids=input_ids, token_type_ids=segment_ids, attention_mask=input_mask)
-    pooled, unpooled, other = output[1], output[0], output[2:]
-    return pooled, unpooled, other
+def get_output_from_standard_transformer_models(
+    encoder, input_ids, segment_ids, input_mask, output_hidden_states=False
+):
+    output = encoder(
+        input_ids=input_ids,
+        token_type_ids=segment_ids,
+        attention_mask=input_mask,
+        output_hidden_states=output_hidden_states,
+    )
+    return output.pooler_output, output.last_hidden_state, output
```

Comment:
@zphang Do you know what the expected contents of `other` is? I'm currently looking at https://huggingface.co/transformers/_modules/transformers/modeling_bert.html. I think I need to make an additional change so that `other` contains the expected values (used here: https://github.com/nyu-mll/jiant/blob/master/jiant/proj/main/modeling/taskmodels.py#L240)

Reply:
Looking at this again...it looks like other should be the `hidden_states`. Let me know if I am missing anything.

Reply:
Hidden states/attention weights. One major benefit of using `return_dict` is to be able to do away with "other".